### ISSUE TITLE
Verification au chargement de la page.

### ISSUE THREAD

**Issue by lion24** [checkmyhttps/checkmyhttps#3]
Hello,

Est-il possible d'effectuer une vérification automatique lorsque le document à été chargé (sans devoir cliquer sur le cadenas). Je crois que en bindant une fonction sur l'evenénement `onload` du DOM on devrait sans sortir, je ne sais pas si cela est déjà implémenté.

Merci.

Hi,

Is it possible to automatically check the status of an HTTPS cert when the document has been loaded (without the need to click on the padlock). I think that binding a function on the `onload` event from the DOM should be ok. I don't know if it's already implemented. 

Thanks.

**Comment by checkmyhttps** [comment]
Hi @lion24,

First versions of CheckMyHTTPS works like that. There was no button, so we checked all pages. This implies that your navigation (hostname of websites) was sent to CheckMyHTTPS server. We thought this could pose confidentiality problems. So we decided to let the user control when he wants to check by clicking on the button.

But we have planned to put this feature back as optional in the next version (in about two weeks).

Thanks!
